utilities/meta.yaml

```diff
@@ -22,6 +22,8 @@ requirements:
     - python
     - iris
     - lxml
+    - owslib
+    - requests
```

Comment:
Not sure.  In OpenSUSE the solution was a tool called (osc) builds and does the version control of the packages.  In our case `obvious-ci` abstracts `conda-build` and `binstar`, and the version control is "optional" (~most of the times~ GitHub).

I can only thing of some hackish solutions where obvious would compare the `meta.yaml` file using some git magic and, if different, increments the build number to triggers a new build.